frontend/src/store/store.ts

```diff
@@ -2200,7 +2200,9 @@ export default new Vuex.Store<IState>({
       ];
```

Comment:
You can also add Skill contract role method check in the array that ends here, pretty much the same way as the ones above. Not sure if that's the same for Skill contract, as it's a bit different, so it's just a reminder, you can ignore this comment